articles/custom-domains/additional-configuration.md

```diff
@@ -123,18 +123,18 @@ If you use Auth0 with a custom domain to issue Access Tokens for your APIs, then
 
 ```js
 app.use(jwt({ 
-  issuer: 'https://YOUR-CUSTOM-DOMAIN',
+  issuer: 'https://<YOUR-CUSTOM-DOMAIN>',
   //code omitted for brevity
 }));
 ```
 
 ## Configure SAML identity providers
 
-If you want to use SAML identity providers (IdPs) with your custom domain, you must get the service provider metadata from Auth0 (such as `https://YOUR-CUSTOM-DOMAIN/samlp/metadata?connection=YOUR-CONNECTION-NAME`). This includes updated **Assertion Consumer Service (ACS) URLs**. Then, you have to manually update this value in your IdP(s). This change to your IdP(s) must happen at the same time as you begin using your custom domain in your applications. This can pose a problem if there are multiple IdPs to configure.
+If you want to use SAML identity providers (IdPs) with your custom domain, you must get the service provider metadata from Auth0 (such as `https://<YOUR-CUSTOM-DOMAIN>/samlp/metadata?connection=<YOUR-CONNECTION-NAME>`). This includes updated **Assertion Consumer Service (ACS) URLs**. Then, you have to manually update this value in your IdP(s). This change to your IdP(s) must happen at the same time as you begin using your custom domain in your applications. This can pose a problem if there are multiple IdPs to configure.
 
 Alternatively, you can use signed requests to fulfill this requirement:
 
-- Once your custom domain is set up, go to [Dashboard > Tenant Settings > Custom Domains](${manage_url}/#/tenant/custom_domains) and download the certificate from the link under the **Sign Request** toggle
+- Download the signing certificate from `https://<TENANT>.auth0.com/pem`. Note that `https://<YOUR-CUSTOM-DOMAIN>.com/pem` will return the same certificate.
```

Comment:
Is the last part true @mmaddex ? That doesn't sound right to me, as we're not serving content at `<YOUR-CUSTOM-DOMAIN>`, so how would we be offering a `/pem` route with a certificate at it?

Reply:
Verified that this works ^

Reply:
sorry... just getting to reading these... glad you figured it out!